### PR TITLE
docs: Add "Scaling New Heights with KEDA: Performance, Extensions, and Beyond" KubeCon talk

### PR DESCRIPTION
**Pull request by tomkerkhove**
Add "Scaling New Heights with KEDA: Performance, Extensions, and Beyond" KubeCon talk next to existing talk with overview.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

### CODE REVIEW

File: layouts/partials/home/video.html [hunk 1]
```diff
@@ -1,19 +1,34 @@
 <section class="section has-background-black has-background-pattern">
-  <div class="container">
+  <div class="container">
```
Comment (JorTurFer) [inline layouts/partials/home/video.html:2]:
```suggestion
  <div class="container">
```

**Comment by JorTurFer** [comment]
Should we add also the video from 2023? https://www.youtube.com/watch?v=Db3UcxXbsvw

**Comment by tomkerkhove** [comment]
Let's pick two and update it here, the rest should move to a "videos" page.

I think it's good to show the most recent video with KEDA 101 overview (current state) but ok with anything

**Comment by JorTurFer** [comment]
Let's use the latest two

File: layouts/partials/home/video.html [hunk 2]
```diff
@@ -3,17 +3,32 @@
...
+
+      <div class="column">
+        <p class="title has-text-white-bis is-spaced">
+          An overview of Kubernetes Event-Driven Autoscaling (KEDA)
+        </p>
+
+        <p class="subtitle has-text-light">
+          From KubeCon Europe 2022
+        </p>
+      </div>
```
Comment (tomkerkhove) [inline layouts/partials/home/video.html:31]:
```suggestion
    <div class="columns is-vcentered">
      <div class="column">
        {{ partial "youtube-embed.html" (dict "youtube_id" "wYQ2cvSj6os") }}
      </div>

      <div class="column">
        <p class="title has-text-white-bis is-spaced">
          Exploring KEDA's Graduation and Advancements in Event-Driven Scaling
        </p>

        <p class="subtitle has-text-light">
          From KubeCon North America 2023
        </p>
      </div>
```

**Review by JorTurFer — APPROVED** [review]

**Pull request merged**